inst/rmarkdown/templates/oup/resources/template.tex

```diff
@@ -248,13 +359,22 @@
 $endif$
 $endif$
 
+\bibliographystyle{$if(namedate)$abbrvnat$else$plain$endif$}
```

Comment:
In usual pandoc template we have 
```suggestion
\bibliographystyle{$if(biblio-style)$$biblio-style$$else$plain$endif$}
```

And it would be in YAML 

````yaml
bibliography: mybib.bib
# biblio-style: abbrvnat # uncomment line to use AUTHOR YEAR FORMAT, or change value to change style
````

I would prefer to keep this way, following generic Pandoc doc rather than having a specific variable `namedate` for this format

Reply:
The problem is that the document class has an option called "namedate" that is separate from the bibliographystyle, which is also given in the default template. I don't fully understand why they did it this way, separating one piece of information into two parts, and I fully agree that it is suboptimal, but that is how it is...

Reply:
The relevant code from the CLS file is:

```latex
\DeclareOption{namedate}{\PassOptionsToPackage{authoryear,round}{natbib}}
```

But in the OUP template it has:

```latex
%USE THE BELOW OPTIONS IN CASE YOU NEED AUTHOR YEAR FORMAT.
%\bibliographystyle{abbrvnat}
```

Reply:
if we are ok to no allow user to set another bibliostyle than these two, then we should keep. Otherwise, we need to find a solution. 

This could be documentation only like 

````
# biblio-style: abbrvnat # use this if you need AUTHOR YEAR FORMAT (oup option `namedate`)
````

Reply:
The OUP template says these two formats should be used. I haven't tried others to see what will happen though. I am OK with just allowing these two options.